src/main/java/org/ibp/api/java/impl/middleware/common/MiddlewareFactory.java

```diff
@@ -527,6 +528,12 @@ public ObservationUnitService getObservationUnitService() {
 		return new ObservationUnitServiceImpl(this.getCropDatabaseSessionProvider());
 	}
 
+	@Bean
+	@Scope(value = "request", proxyMode = ScopedProxyMode.TARGET_CLASS)
+	public ExperimentModelGenerator getExperimentModelGenerator() {
```

Comment:
Why do we need `ExperimentModelGenerator ` in BMSAPI?

Reply:
StudyInstanceServiceImpl needs it.